spec/models/appointment_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe Appointment, type: :model do
+  before(:each) do
+    @user = User.create(name: 'John Doe')
+    @doctor = Doctor.create(name: 'Dr. John Doe', specialization: 'Pediatrics')
+  end
+
+  describe 'associations' do
+    it 'should belong to user' do
+      assc = described_class.reflect_on_association(:user)
+      expect(assc.macro).to eq :belongs_to
+    end
+
+    it 'should belong to doctor' do
+      assc = described_class.reflect_on_association(:doctor)
+      expect(assc.macro).to eq :belongs_to
+    end
+  end
+
+  describe 'validations' do
+    it 'is not valid without a description' do
+      appointment = Appointment.new(description: nil)
+      expect(appointment).to_not be_valid
+    end
+
+    it 'is not valid with a description less than 3 characters' do
```

Comment:
For example, you have grouped all the test for the column description in one it block.